kavm/src/kavm/__main__.py

```diff
@@ -136,12 +139,12 @@ def create_argument_parser() -> ArgumentParser:
     # kompile
     kompile_subparser = command_parser.add_parser('kompile', help='Kompile KAVM', parents=[shared_args])
     kompile_subparser.add_argument(
-        '--definition-dir', dest='definition_dir', type=Path, help='Path to store the kompiled definition'
+        '--definition-dir', dest='definition_dir', type=dir_path, help='Path to store the kompiled definition'
```

Comment:
I think this is causing a problem with the build where command rejects if the file `.build/usr/lib/kavm/avm-llvm/avm-execution-kompiled'`, which seems to not automatically generate, doesn't exist. I was able to get it to run only after manually creating this directory.